.editorconfig

```diff
@@ -12,6 +12,16 @@ insert_final_newline = true
 
 ; .NET Code - almost, but not exactly, the same suggestions as corefx
 ; https://github.com/dotnet/corefx/blob/master/.editorconfig
+dotnet_style_operator_placement_when_wrapping = beginning_of_line

```

Comment:
Make sure the `dotnet` stuff falls under something other than `[*]`. I've used [the Roslyn repo `.editorconfig`](https://github.com/dotnet/roslyn/blob/main/.editorconfig) as a pattern before - not only for starting settings, but ordering. It can get really hard to make sure duplicate conflicting entries aren't added if there isn't some sort of ordering. I think `dotnet` directives in there are under `[*.{cs,vb}]`. In fact, I think if you diff ours vs the Roslyn repo one, it might line up pretty close.

We also can't include `end_of_line` here for the Mac/Linux folks. I think the only time I've seen that used successfully is when it's under `[*.sh]` because it matters, but then it also needs to get coordinated with `.gitattributes` to make sure Git auto line endings match up. Generally best to just not specify line endings if it doesn't matter and let the client deal with it.

A blank line above section headers like `[*.cs]` also helps more easily locate those section headers.

Reply:
Ugh, I didn't add any of the extra changes manually; it was that new-fangled UI editor for editorconfig in VS2022 (which I really don't like); it opened it and must have asserted all this extra stuff. I'll just take it out, I only mean to change the warning level on the naming conventions.

Reply:
You VS users and your fancy tooling. 😆